pkg/controllers/management/auth/globalroles/globalrolebinding_handler.go

```diff
@@ -84,6 +89,7 @@ type globalRoleBindingLifecycle struct {
 	crtbCache         mgmtcontroller.ClusterRoleTemplateBindingCache
 	crtbClient        v3.ClusterRoleTemplateBindingInterface
 	grLister          v3.GlobalRoleLister
+	nsLister          normanv1.NamespaceLister
```

Comment:
Nit: When adding a new field like this we tend to use the Wrangler caches instead of the Norman ones. If you could swap it I would appreciate it.